metricbeat/module/vsphere/datastore_usage/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"vsphere",
+        "name":"datastore_usage",
+        "rtt":44269
+    },
+    "vsphere":{
+        "datastore_usage":{
+            "example": "datastore_usage"
```

Comment:
I'll try to use this in this case:
http://www.virtuallyghetto.com/2017/04/govcsim-neat-incubation-project-vcenter-server-esxi-api-based-simulator.html